packages/design-tokens/webpack/common.js

```diff
@@ -1,16 +1,21 @@
-// shared config (dev and prod)
+const { CleanWebpackPlugin } = require('clean-webpack-plugin')
 const { resolve } = require('path')
 
 module.exports = {
   target: 'node',
   entry: ['./index.ts'],
-  externals: {
-    'styled-components': {
-      commonjs: 'styled-components',
-      commonjs2: 'styled-components',
-      amd: 'styled-components',
+  output: {
+    filename: 'index.js',
+    path: resolve(__dirname, '../dist/cjs'),
+    library: {
+      type: 'commonjs2',
```

Comment:
`output.library.type` is recommended over `output.libraryTarget`: https://webpack.js.org/configuration/output/#outputlibrarytarget